src/CodeFormatter.cs

```diff
@@ -131,31 +111,11 @@ internal static class CodeFormatter
             return new WorkspaceFormatResult(filesFormatted, fileCount, exitCode);
         }
 
-        private static string GetReportFilePath(string reportPath)
-        {
-            var defaultReportName = "format-report.json";
-            if (reportPath.EndsWith(".json"))
-            {
-                return reportPath;
-            }
-            else if (reportPath == ".")
-            {
-                return Path.Combine(Environment.CurrentDirectory, defaultReportName);
-            }
-            else
-            {
-                return Path.Combine(reportPath, defaultReportName);
-            }
-        }
-
-        private static Task<Workspace> OpenFolderWorkspaceAsync(string workspacePath, Matcher fileMatcher, CancellationToken cancellationToken)
+        private static Workspace OpenFolderWorkspaceAsync(string workspacePath, SourceFileMatcher fileMatcher)
```

Comment:
rename to `OpenFolderWorkspace` as it doesn't return a task

Reply:
haha. This is why I need a second set of eyes =)